### ISSUE TITLE
Page up/down (ppage/npage) on osx do not work

### ISSUE THREAD

**Issue by ghost** [CleverRaven/Cataclysm-DDA#19685]
OSX: 10.11.6

I tried playing the experimental sdl build 0.C-6031 and in it and also probably in earlier ones there is a problem with scrolling item description screens which should be possible with the page up/down keys. Now, I don't have a keyboard that has those keys, but it seems possible to emulate them via `fn + arrow key` or `fn + shift + arrow key`. However, this doesn't seem to work in cdda.

Just to make sure, I am talking about screens like this one:

<img width="752" alt="screen shot 2016-12-08 at 20 05 50" src="https://cloud.githubusercontent.com/assets/423471/21023657/bcc3568e-bd81-11e6-83be-2ddae540b71f.png">

P.S.: Related: How would I change those local keybindings? I can't seem to find a way to do so. If I try to change the binding and try to assign `fn + shift + arrow key`I get the following:

<img width="752" alt="screen shot 2016-12-08 at 20 08 08" src="https://cloud.githubusercontent.com/assets/423471/21023721/0ea90354-bd82-11e6-9c4c-8912a9c4feb8.png">

**Comment by openmedi** [comment]
Of course, now that I have written up the issue new data presents itself: I tried to register different keys to scroll through those screens and as it turns out, the problem simply seems to be that no keys scroll the item info up or down.

**Comment by remyroy** [comment]
I'm on 10.11.6 as well and it seems like I can perform the various page up/page down interactions with `Fn`+`↑` and `Fn`+`↓` respectively on build 6050.

Is this still an issue for you? If not, please close it.

**Comment by openmedi** [comment]
Still doesn't work for me (I'm now on 6068). However, using `m` works somehow?

**Comment by patternoia** [comment]
> Still doesn't work for me (I'm now on 6068). However, using m works somehow. EDIT: I should add that it works in other menus just fine, but not in the crafting menu.

I approve this on 10.12, same build 6068

**Comment by dobbry-vechur** [comment]
SDL / MacOS / ImGui has changed a lot. And there's no recent activity. Closing.
